plugins/rich-editor/src/scripts/editor/richEditorClasses.ts

```diff
@@ -355,6 +357,6 @@ export const richEditorClasses = useThemeCache((legacyMode: boolean, mobile?: bo
         paragraphMenuMobile,
         separator,
         position,
-        paragraphMenuPanel,
```

Comment:
I'm thinking this belongs more in `editorFormStyles` in KB at this point (since that refactoring occured).